src/Pact/Native/Ops.hs

```diff
@@ -371,6 +374,15 @@ defTrunc n desc op = defRNative n fun (funType tTyDecimal [("x",tTyDecimal),("pr
               | otherwise = evalError' i "Negative precision not allowed"
           fun i as = argsError i as
 
+defUpCast :: NativeDefName -> Text -> (Integer -> Rational) -> NativeDef
+defUpCast n desc op = defRNative n fun (unaryTy tTyDecimal tTyInteger)
+                     [ ExecExample $ "(" <> asString n <> " 3)"
+                     ]
+                     (desc <> " value of integer X as decimal.")
+    where fun :: RNativeFun e
+          fun _ [TLiteral (LInteger d) _] = return $ tLit $ LDecimal $ fromRational $ op d
+          fun i as = argsError i as
+
```

Comment:
In order to see how this is rendered, you need to build the docs in the `docs` directory with `./build.sh`. I'd like to see how `asString n` is rendered here.